src/main/java/org/terasology/climateConditions/alterationEffects/BodyTemperatureAlterationEffect.java

```diff
@@ -0,0 +1,125 @@
+
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions.alterationEffects;
+
+import org.terasology.alterationEffects.AlterationEffect;
+import org.terasology.alterationEffects.AlterationEffects;
+import org.terasology.alterationEffects.OnEffectModifyEvent;
+import org.terasology.context.Context;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.logic.delay.DelayManager;
+
+/**
+ * This handles the application of the body temperature alteration effect, which modifies the change in body temperature
+ * (based on the magnitude) for a specified duration.
+ */
+public class BodyTemperatureAlterationEffect implements AlterationEffect {
+
+    public static final String BODY_TEMPERATURE = "BodyTemperature";
```

Comment:
This constant should only be defined once. I'm not sure which place would be better, maybe we can make it consistent with the other effects in the AE module.

Reply:
Aren't we keeping this effect separate from all the alteration effects in the AE module?
Since we want BT Alteration Effect only when CC is enabled, then this constant(which indicates the existence of BT effect ) should not be present when the CC module isn't present in the workplace.. So I think.. this should remain here....

Reply:
Sorry, my comment might have been a bit misleading 🙈 the solution you now went for is what I had in mind 👍